packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1521,12 +1521,21 @@ class FocusManager with DiagnosticableTreeMixin, ChangeNotifier {
     if (kFlutterMemoryAllocationsEnabled) {
       ChangeNotifier.maybeDispatchObjectCreation(this);
     }
-    if (kIsWeb || defaultTargetPlatform != TargetPlatform.android) {
+    final bool shouldListenToAppLifecycle = switch (defaultTargetPlatform) {
```

Comment:
@nate-thegrate Sorry but it looks like I missed it in the original review: why's this enabled for `kIsWeb` is true now?

Reply:
Looks like the test is failing too.

Reply:
Good catch, thank you!